src/main/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/FilterClauseDeserializer.java

```diff
@@ -161,7 +161,7 @@ private void validate(
       switch (valueComparisonOperator) {
         case IN -> {
           if (filterOperation.operand().value() instanceof List<?> list) {
-            if (list.size() > operationsConfig.defaultPageSize()) {
+            if (list.size() > operationsConfig.maxInOperatorValueSize()) {
```

Comment:
You were planning to run some test to see how many IN values api could support, any results from that?

Reply:
I will create a microbench ticket for this. Occupied with other things.